src/commands/Miscellaneous/miku.js

```diff
@@ -0,0 +1,44 @@
+const Command = require("../../structures/Command");
+const fetch = require("node-fetch");
+
+class mikuCommand extends Command {
+  constructor(...args) {
+    super(...args, {
+      aliases: ["hatsunemiki"],
+      args: "[tags:string]",
+      description: "Sends an image of Hatsune Miku.",
+      cooldown: 3,
+    });
+  }
+
+  async run(msg) {
+    const body = await fetch("https://safebooru.org/index.php?page=dapi&s=post&q=index&json=1&tags=hatsune_miku%20rating:safe")
+      .then(res => res.json().catch(() => {}));
+
+    if (!body || !body[0].image || !body[0].directory) {
+      return this.bot.embed("❌ Error", "No images were found.", msg, "error");
+    }
+
+    if (body[0].image.endsWith(".webm") || body[0].image.endsWith(".mp4")) {
+      return this.bot.embed("❌ Error", `Post is a video. You can view it [here](${body[0].image}).`, msg, "error");
```

Comment:
Why is this an error? This should be an informational/warning embed instead.

Reply:
The other commands have it called ❌ Error.

This reminds me we should probably be more diverse in our messages and not just say "something bad happened oh no"